LoopKit.xcodeproj/project.pbxproj

```diff
@@ -2587,6 +2742,108 @@
 /* End PBXProject section */
 
 /* Begin PBXResourcesBuildPhase section */
+		1DEE229E24A676A300693C32 /* Resources */ = {
+			isa = PBXResourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				1DEE229F24A676A300693C32 /* carbs_on_board_output.json in Resources */,
```

Comment:
[nit] What is all of this? It doesn't seem relevant to this PR.

Reply:
This is the result of creating a new target

Reply:
ah but somehow it copied all those bundled resources which are not needed.  I'll get rid of them.